src/model/import.ts

```diff
@@ -30,12 +30,10 @@ export class ScannedImport implements Resolvable {
   type: 'html-import'|'html-script'|'html-style'|'js-import'|string;
 
   /**
-   * URL of the import, relative to the document containing the import.
+   * URL of the import, relative to the base directory.
```

Comment:
I think we may want two fields here. `url` used to represent what was actually in the import feature.